fpdf/svg.py

```diff
@@ -845,6 +852,7 @@ def draw_to_page(self, pdf, x=None, y=None, debug_stream=None):
             debug_stream (io.TextIO): the stream to which rendering debug info will be
                 written.
         """
+        self.img_cache = pdf  # Needed to render images
```

Comment:
This is extremely confusing to read.

I see that you let FPDF implement ImageCacheI, so it obviously works. But since the image cache is only ever used internally, wouldn't it make more sense to use a singleton ImageCache instance as an attribute? That would make it immediately recognizable what's actually going on, and not result in such weird assignments like here.

Reply:
> wouldn't it make more sense to use a singleton ImageCache instance as an attribute?

Yes, you are totally right!

I'll do that right now, thank you for wise feedback 🙂